src/ontology/metadata/ordo.yml

```diff
@@ -2,6 +2,8 @@ id: ORDO
 label: Orphanet Rare Disease Ontology
 prefix_map:
   Orphanet: http://www.orpha.net/ORDO/Orphanet_
+prefix_aliases:  # Keys are mondo-ingest preferred prefixes, and values are alternatives
```

Comment:
FYI: How I've decided to reconcile differences between `mondo-ingest` preferred prefixes (e.g. `Orphanet:`) and OAK/bioregistry (e.g. `ORDO:`).